app/controllers/todo_lists_controller.rb

```diff
@@ -0,0 +1,51 @@
+class TodoListsController < ApplicationController
+    skip_forgery_protection
+    # GET /users/:user_id/todo_lists
+    def index
+      user = User.find(params[:user_id]) 
+      todo_lists = user.todo_lists       
+      render json: todo_lists, status: :ok
+    rescue ActiveRecord::RecordNotFound
+      render json: { error: "User not found" }, status: :not_found
+    end
```

Comment:
We can handle the RecordNotFound in the before_action method also. 
Ref: https://github.com/Shova2/TodoApp/pull/2/files#r1851763249